### ISSUE TITLE
Links are always opened in a new tab with no SR direction

### ISSUE THREAD

**Issue by patrickobrien1** [scniro/react-codemirror2#321]
Links that are embedded in markdown format opens a new tab by default and does not announce via aria-labels that a new tab will be opened upon activation of the link. Opens in new tab or window information is not announced by the screen reader. And visual cue also not provided for the visual users.

WCAG Mapping: 3.2.2 On Input

Test tool: JAWS SR, Firefox
